testsuite/gnat2goto/tests/size_integer_and_objects/test.out

```diff
@@ -1,19 +1,70 @@
-+===========================GNAT BUG DETECTED==============================+
-| GNU Ada (ada2goto) Assert_Failure tree_walk.adb:4673                     |
-| Error detected at check_size.adb:1:11                                    |
-| Please submit a bug report; see https://gcc.gnu.org/bugs/ .              |
-| Use a subject line meaningful to you and us to track the bug.            |
-| Include the entire contents of this bug box in the report.               |
-| Include the exact command that you entered.                              |
-| Also include sources listed below.                                       |
-+==========================================================================+
+Standard_Output from gnat2goto check_size:
```

Comment:
I think it is good to update this and remove the crash but I would prefer not having unsupported features in the golden, expected results.

Reply:
Fixed

Reply:
Looks like this test.out still contains the "unsupported feature" report (below)?

Reply:
Oops - missed that one. Should be fixed now by https://github.com/diffblue/gnat2goto/commit/2d836e0e56c80458063a182788a155e1eaf07e5f